### DISCUSSION TITLE
Can't input accented characters

### DISCUSSION THREAD

**Discussion by rodrigoheck** [koolamusic/chakra-ui-autocomplete#96] [General]
First of all, thanks for this great library!

I am having a small problem typing accented characters like à, ê, ã, etc. inside the box. It is splitting the accentuation from the letter. Like ~a instead of ã. I bet there is a quick way to fix that, but I am not finding any answers for it online.

Any idea?

**Comment by koolamusic** [comment]
oh wow this is interesting. could you possible replicate this on a codesandbox and share, let me take a look

**Comment by koolamusic** [comment]
With regards to this, I think this could be coming from the sorting and filtering implementation https://github.com/koolamusic/chakra-ui-autocomplete/blame/8a4d9d4327f4a7f0a1113ac4996cdd6fe7941f4a/src/index.tsx#L46

**Reply by koolamusic** [reply]
Following up on this you can implement a custom `optionsFilterFunc` to handle special character edge cases yourself
